config/locales/en.yml

```diff
@@ -276,6 +278,66 @@ en:
           hint:
             This will help you to identify them and search for them in your trainee records.
             We may also use it if we need to get in touch with you about this trainee.
+  pages:
+    home:
+      heading: Register trainee teacher
+      body:
+        1: Register trainees with the Department for Education and record the outcome of their training
+      new_trainee_link: Add a Trainee
```

Comment:
```suggestion
      new_trainee_link: Add a trainee
```

The green butt has changed

![image](https://user-images.githubusercontent.com/470137/113732158-41ca6500-96f1-11eb-857f-22fa32cc2c5a.png)